docs/source/workflows/predictors.rst

```diff
@@ -214,6 +214,34 @@ The following example illustrates how an :class:`~citrine.informatics.predictors
         }
     )
 
+Simple mixture predictor
+---------------------------------------
+
+Simple mixtures may contain ingredients that are blends of other simple mixtures.
+Along the lines of the example above, hypertonic saline can be mixed with water to form isotonic saline.
+Often, the properties of a hierarchical mixture are strongly associated with its atomic ingredients.
```

Comment:
I notice now that we should probably avoid the word "atomic" lest it be confused with elemental atoms. I'll change to something like "indivisible" -- @latture does that sit well with you?

Reply:
I think "leaf ingredients" would work here.